upup/pkg/fi/cloudup/scalewaytasks/lb_backend.go

```diff
@@ -163,3 +166,25 @@ func (l *LBBackend) RenderScw(t *scaleway.ScwAPITarget, actual, expected, change
 
 	return nil
 }
+
+type terraformLBBackend struct {
+	LBID            *terraformWriter.Literal `cty:"lb_id"`
+	Name            *string                  `cty:"name"`
+	ForwardProtocol *string                  `cty:"forward_protocol"`
+	ForwardPort     *int32                   `cty:"forward_port"`
+}
+
+func (l *LBBackend) RenderTerraform(t *terraform.TerraformTarget, actual, expected, changes *LBBackend) error {
+	tfName := strings.Replace(fi.ValueOf(expected.LoadBalancer.Name), ".", "-", -1)
+	tf := terraformLBBackend{
+		LBID:            expected.TerraformLinkLBID(tfName),
```

Comment:
It is expected that TerraformLink() points to the resource ID. No need to redefine it for each caller.
Could you adjust here and in all similar situations?
```suggestion
		LBID:            expected. LoadBalancer.TerraformLink(),
```

Reply:
Done, I had to keep giving the terraform name as a parameter for the `instance_ip` and `lb_ip` though, because when I tried to put the `Name` in the `terraformInstanceIP` and `terraformLBIP` structs, it got printed in the result and we don't want that.